src/main/java/net/snowflake/ingest/connection/RequestBuilder.java

```diff
@@ -290,6 +316,22 @@ public RequestBuilder(
         this.userAgentSuffix);
   }
 
+  /**
+   * RequestBuilder - this constructor is for testing OAuth purposes only
+   *
+   * @param securityManager - security manager, either JWTManager or OAuthManager
+   */
+  public RequestBuilder(SecurityManager securityManager) {
+    this.port = DEFAULT_PORT;
+    this.scheme = DEFAULT_SCHEME;
+    this.host = DEFAULT_HOST_SUFFIX;
+    this.securityManager = securityManager;
+
+    this.userAgentSuffix = null;
+    this.authType = "OAuth";
+    this.telemetryService = null;
+  }
```

Comment:
This belongs in a test class

Reply:
Is it possible put a constructor of `RequestBuilder` in other class?

Reply:
Just found out we actually don't need this constructor. Removed.